hivemq-getting-started-cloud/build.gradle

```
@@ -0,0 +1,21 @@
+plugins {
+    id 'java'
+}
+
+group 'org.example'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'org.jetbrains', name: 'annotations', version: '16.0.1'
+    compile group: 'com.hivemq', name: 'hivemq-mqtt-client', version: '1.2.2'
+    compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.6.0'
+    testCompile group: 'org.testcontainers', name: 'testcontainers', version: '1.13.0'
```

```suggestion
    compile group: 'com.hivemq', name: 'hivemq-mqtt-client', version: '1.2.2'
```

Please remove unused dependencies.